src/main/io/gps.c

```diff
@@ -98,7 +98,7 @@ static uint16_t fraction3[2];
 gpsSolutionData_t gpsSol;
 uint32_t GPS_packetCount = 0;
 uint32_t GPS_svInfoReceivedCount = 0; // SV = Space Vehicle, counter increments each time SV info is received.
-uint8_t GPS_update = 0;             // it's a binary toggle to distinct a GPS position update
+gpsUpdateToggle_e GPS_update = GPS_DIRECT_TOGGLE_OFF;             // toogle to distinct a GPS position update (directly or via MSP)
```

Comment:
Use `uint8_t` here (`enum` is not packed, it will take 32bits) ... not nice, but that's C
(`#define` will make this problem a bit less visible)

Reply:
```
uint8_t GPS_update = 0;